cmd/create/cluster/cmd.go

```diff
@@ -598,6 +611,20 @@ func run(cmd *cobra.Command, _ []string) {
 		isIAM = !isSTS
 	}
 
+	isHostedCluster := args.hostedClusterEnabled
+	if interactive.Enabled() && cmd.Flags().Changed("hosted-cp") {
+		isHostedCluster, err = interactive.GetBool(interactive.Input{
+			Question: "Deploy a hosted cluster",
+			Help:     cmd.Flags().Lookup("hosted-cp").Usage,
+			Default:  false,
```

Comment:
The default value should be whatever the flag is currently set as, to ensure that the prompt does not override the user-set flag.